### PR TITLE
Update playbook to install and activate openvswitch

### PR DESCRIPTION
**Pull request by mccv1r0**
Update playbook to install and activate openvswitch

Signed-off-by: Michael Cambria <mcambria@redhat.com>

### CODE REVIEW

**Comment by mtnbikenc** [comment]
Failures are not related to changes in this PR.
/test e2e-aws-workers-rhel7

File: roles/openshift_node/defaults/main.yml [hunk 1]
```diff
@@ -75,6 +75,7 @@ openshift_node_support_packages_base:
   - bash-completion
   - vim-minimal
   - nano
+  - openvswitch
```
Comment (dcbw) [inline roles/openshift_node/defaults/main.yml:78]: @mccv1r0 now that I look back at it, we need openvswitch2.13 as the package name.

**Comment by dcbw** [comment]
Superceded by https://github.com/openshift/openshift-ansible/pull/12223

**Pull request closed without merging**